Restore.ps1

```diff
@@ -76,11 +77,33 @@ Function Get-WinNode($arch) {
         $null = mkdir $targetDir
     }
 
+    $targetDirSymbols = "$LayoutRootSymbols\tools\win-$arch"
+    if (!(Test-Path $targetDirSymbols)) {
+        $null = mkdir $targetDirSymbols
+    }
+
     Copy-Item $nodePath $targetDir
+    Copy-Item $nodePath $targetDirSymbols
```

Comment:
Can we also copy the Nix binaries to the symbols package so the symbols package is a superset of the regular one?

Reply:
Great call. Thanks, I missed that.